File: programmers/Lv_1/KimHyunseo/가운데글자가져오기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(string s) {
	int mid = s.size() / 2;
	string answer = "";
	if (s.size() % 2 == 1)
		answer=s[mid];
	else
	{
		answer += s[mid - 1];
		answer += s[mid];
	}
	return answer;
}

int main()
{
	string a = "abcde";
	string b = "qwer";
	cout<<solution(a)<<endl;
	cout << solution(b) << endl;;

}

File: programmers/Lv_1/KimYongho/가운데글자가져오기.java
class Solution {
  public String solution(String s) {
      String answer = "";
      String array[] = s.split("");
      if(array.length % 2 == 0){
          answer += (array[(array.length/2) - 1] + array[array.length/2]);
      }
      else{
          answer += array[array.length/2];
      }
      return answer;
  }
}
